images/installer/Dockerfile.upi.ci.rhel8

```diff
@@ -15,7 +15,8 @@ FROM registry.ci.openshift.org/ocp/4.8:base
 COPY --from=cli /usr/bin/oc /bin/oc
 COPY --from=builder /go/src/github.com/openshift/installer/bin/openshift-install /bin/openshift-install
 COPY --from=builder /go/src/github.com/openshift/installer/upi /var/lib/openshift-install/upi
-COPY --from=builder /go/src/github.com/openshift/installer/data/data/rhcos.json /var/lib/openshift-install/rhcos.json
+# This was removed, see https://github.com/openshift/enhancements/pull/679
+# COPY --from=builder /go/src/github.com/openshift/installer/data/data/rhcos.json /var/lib/openshift-install/rhcos.json
```

Comment:
Why do we need to keep a comment that this was removed, as opposed to just removing it?